libs/dh/market-participant/data-access-api/src/lib/dh-market-participant-edit-actor-data-access-api.store.ts

```diff
@@ -148,16 +134,88 @@ export class DhMarketParticipantEditActorDataAccessApiStore extends ComponentSto
     )
   );
 
+  readonly loadInitialData = this.effect(
+    (routeParams$: Observable<{ organizationId: string; actorId: string }>) =>
+      routeParams$.pipe(
+        switchMap((x) =>
+          forkJoin({
+            gridAreas: this.getGridAreas(),
+            actorAndContacts: this.getActorAndContacts(x),
+          }).pipe(
+            tap((x) =>
```

Comment:
suggestion: give `x` a more meaningful name. E.g. `response`. Object destructuring can be used as well. E.g `({ gridAreas, actorAndContacts })`.

Reply:
fixed @dzhavat